test.sh

```diff
@@ -781,3 +781,52 @@ no_more_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
 zero_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
                  | grep -oE '^  "data": \[\]')
 [ -n "$zero_events" ]
+
+# create a pending payout
+payout=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts \
+       -d amount=1100 \
+       -d currency=eur \
+  | grep -oE 'po_\w+' | head -n 1)
+
+# check correct status
+payout_status=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts/$payout \
+  | grep -oE '"status": "pending",')
+[ -n "$payout_status" ]
+
+# cancel the payout
+curl -sSfg -u $SK: $HOST/v1/payouts/$payout/cancel \
+  -X POST
+
+# check cancel status
+payout_status=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts/$payout \
+  | grep -oE '"status": "canceled",')
+[ -n "$payout_status" ]
+
+# create a paid payout
+payout=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts \
+       -d amount=1100 \
+       -d currency=eur \
+       -d status=paid \
+  | grep -oE 'po_\w+' | head -n 1)
+
+payout_status=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts/$payout \
+  | grep -oE '"status": "paid",')
+[ -n "$payout_status" ]
+
+# create a failed payout
+payout=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts \
+       -d amount=1100 \
+       -d currency=eur \
+       -d status=failed \
+  | grep -oE 'po_\w+' | head -n 1)
+
+payout_status=$(
+  curl -sSfg -u $SK: $HOST/v1/payouts/$payout \
+  | grep -oE '"status": "failed",')
+[ -n "$payout_status" ]
```

Comment:
The first test "create payout, check correct status, cancel it, check different status" makes sense. But in the 2 other ones, it's not very useful to create an object and check it's property `status` just after.

I propose to remove them, because we don't have such basic tests for other classes, and it makes tests a bit heavier to read.

If you *really* want to keep them, they can at least be simplified:
```sh
curl -sg -u $SK: $HOST/v1/payouts \
      -d amount=1100 \
      -d currency=eur \
      -d status=paid

curl -sg -u $SK: $HOST/v1/payouts \
      -d amount=1100 \
      -d currency=eur \
      -d status=failed
```